### ISSUE TITLE
Initial Level / Karen Feedback

### ISSUE THREAD

**Issue by tgroechel** [interaction-lab/MoveToCode#99]
[Doc](https://docs.google.com/document/d/1uIDW1E_Yb-Qu4-OIUwTp65Ks7kdlUmZ_Dh5c8-AVTGM/edit)
This depends on #98 being done -> this is done
- [x] Solution for snap collider overlay the real blocks -> likely won't fix due to scaling issues/time needed
- [x] [Unique ID per device](https://docs.unity3d.com/ScriptReference/SystemInfo-deviceUniqueIdentifier.html)
- [x] Logging Object manipulator as well as manip handler
- [x] color all around BK plates
- [x] Better play/pause buttons, possible experiment with them on screen
- [x] bk cannot be child of maze pieces, should just be separate so no warping
- [x] lock maze button does not work correctly
- [x] More defined highlights while tracking
- [ ] reset maze piece places (like trash corner or remove)
- [ ] Condense initial screen
- [ ] Log code nicely
- [x] Match color of datablock to color of normal block (might be remedied by scaffolding)

**Comment by tgroechel** [comment]
#101 contains this now, closing
